packages/dds/tree/src/test/snapshots/snapshotTestScenarios.ts

```diff
@@ -333,19 +333,20 @@ export function generateTestTrees(options: SharedTreeOptions) {
 			name: "nested-sequence-change",
 			runScenario: async (takeSnapshot) => {
 				const sf = new SchemaFactory("test trees");
-				class SequenceMap extends sf.mapRecursive("Recursive Map", [
-					() => sf.array(SequenceMap),
+				class Array extends sf.arrayRecursive('Array<["test trees.Recursive Map"]>', [
```

Comment:
While experimenting with some related changes (which I didn't include in this PR), I found that this test is doing the not supported pattern of mixing recursive and non recursive schema in a co-recursive context, so I updated it to have the exact same stored schema (so the snapshots would match) but use the explicitly named recursive schema APIs.